docs/shared/recipes/plugins/create-package.md

```diff
@@ -0,0 +1,75 @@
+
+- The plugin package
+- The create package
+
+You should also have a generator called `preset` that was generated within the plugin package. This preset generator will be invoked when running either `npx create-nx-workspace --preset my-plugin` or `npx create-my-plugin`. For more information about customizing your preset, see: [Creating a Preset](/plugins/recipes/create-preset).
+
+Let's take a look at the code that was scaffolded out for your create-my-plugin package:
+
+```typescript {% fileName="packages/create-my-plugin/bin/index.ts" %}
+#!/usr/bin/env node
+
+import { createWorkspace } from 'create-nx-workspace';
+
+async function main() {
+  const name = process.argv[2]; // TODO: use libraries like yargs or enquirer to set your workspace name
+  if (!name) {
+    throw new Error('Please provide a name for the workspace');
+  }
+
+  console.log(`Creating the workspace: ${name}`);
+
+  // This assumes "my-plugin" and "create-my-plugin" are at the same version
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const presetVersion = require('../package.json').version;
+
+  // TODO: update below to customize the workspace
+  const { directory } = await createWorkspace(`my-plugin@${presetVersion}`, {
+    name,
+    nxCloud: false,
+    packageManager: 'npm',
+  });
+
+  console.log(`Successfully created the workspace: ${directory}.`);
+}
+
+main();
+```
+
+By utilizing your custom preset to handle the generation, we can provide equal functionality between `create-nx-workspace` and your new create-package.
+
+This is a plain node script at this point, and you can use any dependencies you wish to handle things like prompting or argument parsing. Keeping dependencies small and splitting out the command line tool from the Nx plugin is recommended, and will help keep your CLI feel fast and snappy.
+
+## Testing your create package
+
+Because your create-my-plugin package will install your plugin package at runtime, they must be published in order to run them and see the results. In order to test your packages without making them publicly available, a `local-registry` target should be present on project in your workspace's root.
+
+This target will start a local registry server using verdaccio, and configure your machine to point to it. This means that you can safely publish, without hitting npm, and test as if you were an end user of your package. The flow should look something like this:
+
+1. Run `nx local-registry` and wait to see the text that is similar to "http address - http://localhost:${port}/" at this point, you should be able to visit that url and verify that the server is running.
+2. In a second terminal, publish your packages. The exact command here will change depending on your setup, but it could look something like `nx run-many -t publish`. You may wish to verify that you are not publishing to the actual npm registry, by running `npm config get registry` first.
```

Comment:
Split "You may wish to verify..." into its own step that goes before this one.  What should they look for as output from `npm config get registry`?